enterprise/config/buildbuddy.local.yaml

```diff
@@ -13,8 +13,58 @@ storage:
     root_directory: /tmp/buildbuddy_enterprise
   enable_chunked_event_logs: true
 cache:
+  # Sample flow if migration disk -> pebble
+
```

Comment:
Since in the config all of this is parented under the "cache:" section -- would we just always create a cache migrator wrapper around whatever cache is configured?

Reply:
My thought was that we always initialize an empty MigrationCache object on the env, but it there's no active migration happening and its src and dest fields aren't set, main.go would just ignore it and continue setting a cache (with no wrapper) like it currently does

Reply:
See Line #197 in disk_cache.go for kind of a demonstration of what I mean